SPL/SPL.php

```diff
@@ -1097,8 +1097,8 @@ class RegexIterator extends FilterIterator {
      * Create a new RegexIterator
      * @link https://php.net/manual/en/regexiterator.construct.php
      * @param Iterator $iterator The iterator to apply this regex filter to.
-     * @param string $regex The regular expression to match.
-     * @param int $pattern [optional] Operation mode, see RegexIterator::setMode() for a list of modes.
+     * @param string $pattern [optional] The regular expression to match.
```

Comment:
Could you please clarify why this parameter is marked as optional?

Reply:
The parameter "$regex" is not available in the __constructor, it was renamed from "$regex" to "$pattern"

Reply:
The question is why `$pattern` is marked as `[optional]`? As I can see in docs and PHP src this parameter isn't optional. `$regex` previously also was not optional

Reply:
Sorry, you are right... was a copy&past error.